src/app.types.tsx

```diff
@@ -38,36 +38,31 @@ export interface CatalogueCategory {
   is_leaf: boolean;
   catalogue_item_properties?: CatalogueCategoryFormData[];
 }
+
 export interface AddManufacturer {
   name: string;
   url?: string;
```

Comment:
I double checked, it was fixed here https://github.com/ral-facilities/inventory-management-system-api/commit/f576bbf01d8b4cd708696e85e019c26fb7db0f4a so could technically be `| null` if we wanted now.
```suggestion
  url?: string | null;
```